src/Components/Counter.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState, useEffect } from 'react';
+import { db } from '../lib/firebase';
+
+const Counter = () => {
+  const [count, setCount] = useState();
+  const newCount = db.collection('count').doc('FQyeXUHAfge9AXp6qXrK');
+
+  useEffect(() => {
+    newCount.get().then(function (doc) {
+      setCount(doc.data().number);
+    });
+  }, [newCount]);
+
+  const increaseCount = (newCount) => {
+    setCount(count + 1);
+    newCount.update({ number: count + 1 });
+  };
```

Comment:
There might be a case where the update to the Firestore might not go through and the counts will be out of sync. You can try updating the Firestore first and then use `then` to update the browser count if it is successful and `catch` to log any errors if it isn't successful.

Reply:
Thanks for the feedback! We added `then` and `catch` to both our get request and our update request to catch any errors that we may have!